### ISSUE TITLE
The focus indicator is not clearly on 'Clear column filter', 'Text filter' submenu, controls in submenu and 'search field' present under 'Name' filter.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7233]
**Test Environment:**
OS: Windows_11
Version: 21H2
OS Build: 22000.795
Browser: Version 103.0.1264.71 (Official Build) (64-bit)
URL: https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering

**Repro Steps:**
1. Open [Angular Grid | Excel Style Filtering | Infragistics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering)
2. Grids excel style filtering pen page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Press ctrl + shift + L, the context menu opens.
6. With 3-tab press reach to the filter control and activate it.
7. Verify the issue that focus is visible properly on Filter pane controls.

**Actual Behavior:**
The focus indicator is not clearly visible on the 'Clear column filter', 'Text filter' submenu, controls in the submenu, and 'search field' present under the 'Name' filter.

**Expected Behavior:**
The focus indicator should be clearly visible on the 'Clear column filter', 'Text filter' submenu, controls in the submenu, and 'search field' present under the 'Name' filter.
<img width="958" alt="MAS2 4 7_focus indicator not visible" src="https://user-images.githubusercontent.com/93735775/183061269-790a3f4b-aa9a-4401-a424-28a1d9615b38.png">

<img width="960" alt="MAS2 4 7-MCIO DCX-Name filter 2" src="https://user-images.githubusercontent.com/93735775/183061293-bbaf30e9-3f31-4548-a06b-5530047f7c4f.png">

**Comment by mportuga** [comment]
This defect is not for UI-Grid, but for Ignite UI, which is managed by a completely different team.
